docs/configuration-files.md

```diff
@@ -271,6 +272,117 @@ rules:
 
 This rule looks for usage of Django's [`FloatField`](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.FloatField) model when storing currency information. `FloatField` can lead to rounding errors and should be avoided in favor of [`DecimalField`](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.DecimalField) when dealing with currency. Here the `pattern-where-python` operator allows us to utilize the Python `in` statement to filter findings that look like currency.
 
+## Metavariable matching
+
+### Metavariables in logical inclusions
```

Comment:
The rest of this document uses "logical AND" and "logical OR" vs. "logical inclusion" and "logical union". I also think that language is simpler to understand for an entry-level engineer vs. a program analysis expert.

Reply:
I have inclusion here to be inclusive (heh) of both `patterns` and `pattern-inside`, but maybe we should just be extensional.